jenkins/pipelines/QE/sgw/test.sh

```diff
@@ -44,11 +42,7 @@ fi
 
 # Run Tests :
 echo "Run tests..."
-
-pushd "${QE_TESTS_DIR}" > /dev/null
-create_venv venv
-. venv/bin/activate
-pip install -r requirements.txt
-pytest -v --no-header -W ignore::DeprecationWarning --config config.json -m sgw
-deactivate
+pushd $QE_TESTS_DIR > /dev/null
+uv sync --no-install-project
+uv run --no-project pytest -v --no-header -W ignore::DeprecationWarning --config config.json -m sgw test_xattrs.py
```

Comment:
I'm confused about this.  If you say "no project" won't this ignore the venv you just set up with sync?